huaweicloud/services/dli/resource_huaweicloud_dli_queue.go

```diff
@@ -357,6 +365,20 @@ func resourceDliQueueUpdate(ctx context.Context, d *schema.ResourceData, meta in
 		}
 	}
 
+	oldVal, newVal := d.GetChange("elastic_resource_pool_name")
+	if oldVal == "" && newVal != "" {
```

Comment:
just use `if oldVal == "" {...}else{...}` will be fine

Reply:
fixed